schedule/yast/autoyast/autoyast_ext4.yaml

```diff
@@ -5,7 +5,10 @@ description: >
   with product defined in the profile.
 vars:
   AUTOYAST: autoyast_sle15/autoyast_ext4.xml
+  AUTOYAST_PREPARE_PROFILE: '1'
   DESKTOP: gnome
+  FILESYSTEM: ext4
+  INSTALLATION_VALIDATION: autoyast/verify_ext4
```

Comment:
Hope it will be not to annoying to remove some variables which we used for scheduling. `INSTALLATION_VALIDATION` is one of those, so we can get rid of it.

Reply:
Should I remove INSTALLONLY and INSTALLATION_VALIDATION from all schedules in schedule/yast/ ?

Reply:
That would be nice, if you don't mind. But not adding them to these scenarios is more than sufficient too.

Reply:
Done